RecoLocalCalo/HcalRecProducers/python/HcalHitReconstructor_ho_cfi.py

```diff
@@ -31,7 +32,11 @@
     setTimingTrustFlags = cms.bool(False), # timing flags currently only implemented for HF
     setPulseShapeFlags = cms.bool(False),  # not yet defined for HO
     setNegativeFlags          = cms.bool(False),  # only in HBHE
-    saturationParameters=  cms.PSet(maxADCvalue=cms.int32(127))
+    saturationParameters=  cms.PSet(maxADCvalue=cms.int32(127)),
+    # Configuration parameters for Method 3
+    pedestalSubtractionType = method3.pedestalSubtractionType,
```

Comment:
the framework will put the default values defined with fillDescription from the HcalHitReconstructor .
The parameters are bogus anyways (nothing to be set for HF or HO hits) as an aux energy setting parameters.

Even if we will have M3 for HO or HF, I doubt very much that the HBHE-derived values will apply.
I suggest to remove to reduce confusion.